release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -72,6 +71,27 @@ Previously, secret information was used as the OAuth access token and OAuth auth
 If you are upgrading your cluster to {product-title} 4.6, old tokens from {product-title} 4.5 will still have the secret information exposed in the object name. By default, the expiration for tokens is 24 hours, but this setting can be changed by administrators. Sensitive data can still be exposed until all old tokens have either expired or have been deleted by an administrator.
 ====
 
+[id="ocp-4-6-scale"]
+=== Scale
+
+[id="ocp-4-6-scale-cluster-maximums"]
+==== Cluster maximums
+
+Updated guidance around
+xref:../scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc#planning-your-environment-according-to-object-maximums[Cluster
+maximums] for {product-title} {product-version} is now available.
+
+Use the link:https://access.redhat.com/labs/ocplimitscalculator/[{product-title}
+Limit Calculator] to estimate cluster limits for your environment.
+
+[id="ocp-4-6-scale-real-time-profile-added-to-node-tuning-operator"]
+==== Real-time profile added to the Node Tuning Operator
+
+Partial tuned real-time profile support became available in {product-title} 4.4. However, a lot of
+node-management tasks still needed to be performed by the administrator or by an
+add-on Operator. Now, profile delivery is fully automated and performed from a
```

Comment:
Unfortunately, this is not quite true.  There are many aspects to real-time tuning.  We/Customers might still want to use an addon operator (e.g. "PAO") to take care of some extra OCP-specific tuning aspects -- there are many of them.  What changed, however, is that the realtime profiles shipped in OCP 4.6 are fully compatible with what the realtime profiles do in Tuned on RHEL now.  And this is what NTO is trying to do at the moment, no extra tuning, such as setting kubelet parameters and so on.

Reply:
@jmencak Thanks! Updated. PTAL: http://file.rdu.redhat.com/~ahardin/Sept092020/real-time-profile-nto/release_notes/ocp-4-6-release-notes.html